common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -5508,6 +5508,17 @@ public static enum ConfVars {
     LLAP_TASK_TIME_SUMMARY(
         "hive.llap.task.time.print.summary", false,
         "Display queue and runtime of tasks by host for every query executed by the shell."),
+
+    LLAP_LOCAL_DIR_CLEANER_CLEANUP_INTERVAL(
+        "hive.llap.local.dir.cleaner.cleanup.interval", "7200s", new TimeValidator(TimeUnit.SECONDS),
+      "Interval by which the LocalDirCleaner service in LLAP daemon checks for stale/old files. \n" +
+      "Under normal circumstances, local files are cleaned up properly, so it's not recommended to \n" +
+      "set this more frequent than a couple of hours. Default is 2 hours."),
+    LLAP_LOCAL_DIR_CLEANER_FILE_MODIFY_TIME_THRESHOLD("hive.llap.local.dir.cleaner.file.modify.time.threshold", "86400s",
```

Comment:
Nit: If the usual period is at least one day, can we use hour as the default unit?
`LLAP_LOCAL_DIR_CLEANER_FILE_MODIFY_TIME_THRESHOLD("hive.llap.local.dir.cleaner.file.modify.time.threshold", "24h", new TimeValidator(TimeUnit.HOURS),`

Reply:
makes sense, I can change it to hour as default unit